page-install.php

```diff
@@ -9,8 +9,9 @@
       <a class="btn btn-default btn-lg" href="#instructions-web" role="button" data-toggle="popover" rel="tooltip" title="Best for shared hosts"><i class="icon-code"></i> Web Installer</br><small>For shared hosts</small></a>
       <a class="btn btn-default btn-lg"  href="#instructions-packages" role="button" data-toggle="popover" rel="tooltip" title="Provides automated updates"><i class="icon-linux"></i> Packages</br><small>For auto updates</small></a>
     </div>
-    <p>Looking for <a href="/changelog">older versions</a>?<br />
-    Want to use <a target="_blank" href="https://owncloud.com/products/enterprise/">ownCloud for the Enterprise?</a></p>
+    <p>Looking for <a href="/changelog">older versions</a>?
+<!--     <br /> Want to use <a target="_blank" href="https://owncloud.org/support">ownCloud professionally?</a> -->
```

Comment:
Let's just remove this, we have git so it's always in the history. Otherwise we end up with all this lying around.